public/app/plugins/panel/logs/LogsPanel.tsx

```diff
@@ -70,6 +72,8 @@ export const LogsPanel: React.FunctionComponent<LogsPanelProps> = ({
   useLayoutEffect(() => {
     if (isAscending && logsContainerRef.current) {
       setScrollTop(logsContainerRef.current.offsetHeight);
+    } else if (logsContainerRef.current) {
+      setScrollTop(logsContainerRef.current.scrollHeight);
```

Comment:
If new entries are added at the top of the log (or the bottom, if we decide to have `new` at the bottom the same way the current editor does), will this still work correctly?